src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -220,6 +220,18 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             status: None,
         };
 
+        let volume_name = "secrets-mount".to_string();
+        let secret_name = "dataflowd-secret".to_string();
```

Comment:
This should somehow reuse the constant from the secrets controller: https://github.com/MaterializeInc/materialize/blob/ec558d7aed414e8980b787112e32d3c539540989/src/secrets-kubernetes/src/lib.rs#L23.

Reply:
That name leaves a bit to be desired too.

Reply:
What would you guys prefer?

Reply:
We want you to `use` the secret from the secrets-kubernetes lib, but I'm not a huge fan of the name of that constant. Even `SECRET_NAME` would be better.